contracts/Vault.vy

```diff
@@ -116,6 +116,46 @@ event StrategyReported:
     debtLimit: uint256
 
 
+event SetName:
+    name: String[64] # New active Vault ERC20 name
+
+
+event SetSymbol:
+    symbol: String[32] # New active Vault ERC20 symbol
+
+
+event SetGuestList:
+    guestList: indexed(address) # Vault guest list address
+
+
+event SetRewards:
+    rewards: indexed(address) # New active rewards recipient 
+
+
+event SetDepositLimit:
+    depositLimit: indexed(uint256) # New active deposit limit
+
+
+event SetPerformanceFee:
+    performanceFee: indexed(uint256) # New active performance fee
+
+
+event SetManagementFee:
+    managementFee: indexed(uint256) # New active management fee
+
+
+event SetGuardian:
+    guardian: indexed(address) # Address of the active guardian
```

Comment:
No need to index these

Only index `strategy` address in `UpdateStrategy*` events